check-requirements.in

```diff
@@ -1,7 +1,16 @@
 bandit
+darglint
 flake8
+flake8-annotations-complexity
+flake8-bugbear
+flake8-broken-line
+flake8-commas
```

Comment:
Expect trailing commas basically everywhere where something spans multiple lines.

Reply:
O this one is especially nice. Git will love us.

Reply:
Yeah exactly, especially as it wasn't reporting things that neatly fit on a single line, top implementation there :ok_hand: